.github/workflows/tests_post.yml

```diff
@@ -0,0 +1,34 @@
+name: Post Tests
+# This is for tests that run on a PR post merge. This to optimize CI test suite time
+# for exceptionally long running test processes. Errors will still generate an
+# issue against the PR/commit so will be caught.
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+
+  xtensa_postmerge:
+    uses: ./.github/workflows/xtensa_postmerge.yml
```

Comment:
add check that the pull_request was merged?

Reply:
Added merge check, removed logic around dispatched and scheduled runs, moved some jobs between presubmit and postmerge as per conversation last week with @advaitjain. Have updated list of branch protection tests in the description to match the new test structure.